engine/src/test/resources/testTopology.json

```diff
@@ -25,7 +25,7 @@
           "attributes": {
             "UNIFIER_LIMIT": 8,
             "STREAM_CODEC" : {
-              "com.datatorrent.common.codec.JsonStreamCodec" : {}
+              "com.datatorrent.stram.plan.logical.LogicalPlanConfigurationTest$TestStreamCodec" : {}
             }
```

Comment:
I Tested with the original JSON and the testLoadFromJson test failed because the attribute is not serializable. As it is now this PR is backwards incompatible. Could we make the JsonStreamCodec serializable to make the original JSON work or does that introduce other issues?